tutorials/2_Gazebo_and_ROS.ipynb

```diff
@@ -0,0 +1,1330 @@
+    "gedit diff_wheeled_robot.xacro"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "<?xml version=\"1.0\"?>\n",
+    "<robot name=\"mybot\" xmlns:xacro=\"http://www.ros.org/wiki/xacro\">\n",
+    "        <!-- Put here the robot description -->\n",
+    "</robot>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>The structure of a URDF file is relatively simple. The complete description (links, joints, transmission, etc.) has to be within the robot tag. The xmlns:xacro=\"http://www.ros.org/wiki/xacro\" specifies that this file will use xacro. With xacro, you can define parameters. This will make the file clearer. They are usually put at the beginning of the file (within the robot tag, of course). We will not use xacro for now, but you can find more details about xacro <a href:\"http://wiki.ros.org/urdf/Tutorials/Using%20Xacro%20to%20Clean%20Up%20a%20URDF%20File\">here</a>.</p>\n",
+    "\n",
+    "<p>Let’s define some physical properties for our robot, mainly the dimensions of the chassis, the caster wheel, the wheels and the range sensor:</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "  \n",
+    "<!-- PROPERTY LIST -->\n",
+    "  <!--All units in m-kg-s-radians unit system -->\n",
+    "  <property name=\"M_PI\" value=\"3.1415926535897931\" />\n",
+    "  <property name=\"M_PI_2\" value=\"1.570796327\" />\n",
+    "  <property name=\"DEG_TO_RAD\" value=\"0.017453293\" />\n",
+    "\n",
+    "  <!-- Main body radius and height -->\n",
+    "  <!-- Main Body Cylinder base   -->\n",
+    "  <property name=\"base_height\" value=\"0.02\" /> \n",
+    "  <property name=\"base_radius\" value=\"0.15\" /> \n",
+    "  <property name=\"base_mass\" value=\"5\" /> <!-- in kg-->\n",
+    "\n",
+    "\n",
+    "\n",
+    "  <!-- caster wheel radius and height -->\n",
+    "  <!-- caster wheel mass -->\n",
+    "  <property name=\"caster_f_height\" value=\"0.04\" /> \n",
+    "  <property name=\"caster_f_radius\" value=\"0.025\" /> \n",
+    "  <property name=\"caster_f_mass\" value=\"0.5\" /> <!-- in kg-->\n",
+    "\n",
+    "\n",
+    "\n",
+    "  <!-- caster wheel radius and height -->\n",
+    "  <!-- caster wheel mass -->\n",
+    "  <property name=\"caster_b_height\" value=\"0.04\" /> \n",
+    "  <property name=\"caster_b_radius\" value=\"0.025\" /> \n",
+    "  <property name=\"caster_b_mass\" value=\"0.5\" /> <!-- in kg-->\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Every file must contain the robot tag and everything we put in them should be in this tag.</p>\n",
+    "\n",
+    "<p>Now we want to add a circular base for our robot. Insert this within the robot tag of diff_wheeled_robot_description.xacro:</p>"
```

Comment:
Like I said before, show a picture of the robot they are building at the start of the section because at this point it is not clear why we are adding a circular base